index.js

```diff
@@ -155,6 +155,9 @@ function getEncodingHeader (request) {
     if (supportedEncodings.indexOf(acceptEncodings[i]) > -1) {
       return acceptEncodings[i]
     }
+    if (acceptEncodings[i].indexOf('*') > -1) {
```

Comment:
I think this can be faster.
If a user puts `*` as accepted encodings, there will not be other stuff there.
if so, we can just check it the header value is equal to `*`.

Does it make sense?

Reply:
No, we cannot do that. Something like 'gzip;q=1.0, *;q=0.5' is valid and might occur. This would mean use gzip if possible and else use any other encoding.